### PR TITLE
Style app

### PR DESCRIPTION
**Pull request by Sahar-AbdelSamad**
In this PR I:
- styled the app following the Bookstore design in [Zeplin](https://zeplin.io/)
- Updated the README file

## Important note for the Code reviewer 
- My project works perfectly on my local machine with no linter errors, but when I push it to GitHub, I get a lot of eslint errors
- There is a problem with `eslint-plugin-react` in its latest version (published 2 hours ago) (https://github.com/vercel/next.js/discussions/34797), so my First PullRequest was invalidated and the second was rejected because of it
- I removed `eslint` from the workflow folder and from my project, you can check that it is free of linter errors locally
- Thank you for taking the time to review my code

### CODE REVIEW

**Review by elyor-doniyorov — COMMENTED** [review]
Hi @Sahar-AbdelSamad, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/assets/BookList.css [hunk 1]
```diff
@@ -1,43 +1,200 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab:wght@700&display=swap');
+
```
Comment (elyor-doniyorov) [inline src/assets/BookList.css:2]: - [x] Your app looks awesome. You made your app styling exactly the same as provided on Zeplin. Good job! However, when running the npm start I am having an error 👇 
Screenshot (207)
Kindly fix it before requesting for another review 🙏

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @Sahar-AbdelSamad, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] Branch name does not match the current milestone, kindly update it.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -2,7 +2,13 @@
...
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-> The Bookstore is a website that allow us to `Display a list of books`, `Add a new book` and `Remove a selected book`.
+> The Bookstore is a React-Redux application connected to an existing API to create and remove books in a remote server. Users can **Display the list of books**, **Add** and **Remove a selected book**.
+
+![screenshot](./Screenshot.gif)
+
+## Live Demo
+
+You can see [the app live here](https://sahar-abdelsamad.github.io/bookstore/)
```
Comment (DeliceLydia) [inline README.md]: - [x] Fix your live demo link

File: src/assets/BookList.css (hunk 1 above)
Comment (DeliceLydia) [reply src/assets/BookList.css:2]: resolved

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Sahar-AbdelSamad, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Sahar-AbdelSamad) [reply README.md]: Done :+1:

File: src/assets/BookList.css (hunk 1 above)
Comment (Sahar-AbdelSamad) [reply src/assets/BookList.css:2]: Fixed :+1:

**Review by mmsesay — APPROVED** [review]
Hi @Sahar-AbdelSamad, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**